### PR TITLE
EZP-27532 translations tab

### PR DESCRIPTION
**Pull request by mcurran-inviqa**
https://jira.ez.no/browse/EZP-27532
image

### CODE REVIEW

**Comment by mcurran-inviqa** [comment]
https://jira.ez.no/browse/EZP-27532
image

**Review by maxbaldanza — APPROVED** [review]
👍 for going back and adding `desc()` to the tabs

**Comment by maxbaldanza** [comment]
Probably not for this ticket but putting this out there for suggestions: what are your thoughts on splitting up these actions in to separate controllers so they all handle one single responsibility?
That way the controller doesn't become over 200 lines of code and we can follow something close to the ADR (action domain responder) pattern

File: src/bundle/Resources/views/locationview.html.twig [hunk 1]
```diff
@@ -14,6 +14,7 @@
             <li class="ez-tabs-label"><a href="#ez-tab-versions">{{ 'tab.versions'|trans|desc('Versions') }}</a></li>
             <li class="ez-tabs-label"><a href="#ez-tab-locations">{{ 'tab.locations'|trans|desc('Locations') }}</a></li>
             <li class="ez-tabs-label"><a href="#ez-tab-relations">{{ 'tab.related.content'|trans|desc('Related content') }}</a></li>
+            <li class="ez-tabs-label"><a href="#ez-tab-translations">{{ 'tab.translations'|trans()|desc('Translations') }}</a></li>
```
Comment (bdunogier) [inline src/bundle/Resources/views/locationview.html.twig]: You can skip the `()` in the `trans` call.

**Review by bdunogier — APPROVED** [review]
Good to go.

**Pull request merged**